simulators/nav/src/store/modules/roverState.ts

```diff
@@ -38,11 +52,32 @@ const state:RoverState = {
     total_wps: 0
   },
 
+  noiseSetttings: {
+    locNoise: {
+      headingStdev: 5,
+      latLonStdev: 1
+    },
+    percepNoise: {
+      obsFalsePos: 0.1,
+      obsFalseNeg: 0.1,
+      tagFalsePos: 0.1,
+      tagFalseNeg: 0.1,
+      tagIdFalses: 0.1,
+      bearStddev: 5,
+      distStddev: 0.5
+    }
```

Comment:
@mmartin4972 Can you help me choose reasonable values for these?

Reply:
We've never thrown a FP for tags, since our AR tag stuff is super picky. I'd set **tagFalsePos** to 0, or 0.01 if you still want to account for it. Since we're picky set **tagFalseNeg** to 0.15. Not sure if you account for us getting more accurate as we move closer to the waypoint, but on average I'd say 0.15 is probly a better representation. Set **obsFalseNeg** to 0.05. It's more likely that we see an obstacle that isn't actually there than don't see an obstacle that is there. 

We have ways to tune our code to give you more FP and fewer FN or vice versa, so after testing w the sim if you find you prefer more FN than FP or something like that lmk and we can tune our code accordingly

Reply:
I adjusted the numbers to reflect your suggestions. I wanted to avoid 0 so I went with 1%. I attempted to implement getting more accurate as we got closer but it turned out that would be pretty complicated so I opted to put that off for now. That's definitely something that can be looked into in the future though.